test/elixir_search_extractor_web/controllers/api/v1/keyword_file_controller_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule ElixirSearchExtractorWeb.Api.V1.KeywordFileControllerTest do
+  use ElixirSearchExtractorWeb.ConnCase, async: true
+
+  import ElixirSearchExtractor.{AccountsFixtures, AccessTokenFixture, KeywordFileFixtures}
+
+  describe "post api/v1/keyword_files" do
+    test "returns 201 status with 'Upload Successful' message when valid data is given", %{
+      conn: conn
+    } do
+      user = user_fixture()
+
+      conn =
+        conn
+        |> set_authentication_header(user)
+        |> post(Routes.api_v1_keyword_file_path(conn, :create), valid_keyword_file_attributes())
+
+      assert conn.status == 201
+      assert conn.resp_body == "Upload Successful"
```

Comment:
Not sure if there is any format for success message in json API 🤔

Reply:
I think I should have sent the created file instead of a message. Added in [1dbb532](https://github.com/rafayet-monon/elixir_search_extractor/pull/53/commits/1dbb53226c9e26174e5dbcb4a684df5fd02418b7) @rosle